server/aws-lsp-codewhisperer/src/language-server/inline-completion/codeWhispererServer.ts

```diff
@@ -362,11 +381,38 @@ export const CodewhispererServerFactory =
                             extraContext + '\n' + requestContext.fileContext.leftFileContent
                     }
 
-                    const generateCompletionReq = {
-                        ...requestContext,
-                        ...(workspaceId ? { workspaceId: workspaceId } : {}),
+                    // Prepare the request with normalized file content
+                    const normalizedFileContext = {
+                        ...requestContext.fileContext,
+                        leftFileContent: requestContext.fileContext.leftFileContent
+                            .slice(-CONTEXT_CHARACTERS_LIMIT)
+                            .replaceAll('\r\n', '\n'),
+                        rightFileContent: requestContext.fileContext.rightFileContent
+                            .slice(0, CONTEXT_CHARACTERS_LIMIT)
+                            .replaceAll('\r\n', '\n'),
```

Comment:
it's already handled by https://github.com/aws/language-servers/blob/main/server/aws-lsp-codewhisperer/src/shared/codeWhispererService.ts#L94-L100 ?